src/index.ts

```diff
@@ -9,3 +9,37 @@ export {
    Response,
    Router,
 };
+
+// We need to export only types that are used in public interfaces (e.g. those used in
```

Comment:
FWIW, I doubled-checked this list against the output from [Microsoft's API Extractor tool](https://api-extractor.com/) to ensure that all of the types used in the public API, but I opened https://github.com/silvermine/lambda-express/issues/49 to make considering integrating that tool a separate task. Integrating it will not be simple, and I'm not certain it will even completely meet our needs (it doesn't support literal `type` definitions, etc).